CargoWebAsset.js

```diff
@@ -46,10 +50,10 @@ class CargoWebAsset extends Asset {
         }
     }
 
-    static async install_cargo_web() {
+    async install_cargo_web() {
         let install_required;
         try {
-            const cargo_web_version = await execFile( "cargo-web", [ "--version" ]);
+            const cargo_web_version = await execFile( this.cargo_web_command(), [ "--version" ]);
```

Comment:
One thing that seems brittle here, consider this scenario:

- User has `CARGO_WEB` defined, but it's invalid for whatever reason.
- We execute `$CARGO_WEB -- version` which fails
- We install cargo web
- The environment variable is still invalid, and subsequent calls to cargo web will fail

Any ideas on the best way to handle this? Redefining `$CARGO_WEB after an install seems like it would defy the user's expectations. Perhaps just throw an error?

Reply:
Hmm... well, if `CARGO_WEB` is defined I guess we don't automatically install a new version and if the version itself doesn't match our minimum requirements we just error out? That would seem reasonable to me.